dockernet/ts-tests/test/main.test.ts

```diff
@@ -0,0 +1,180 @@
+    const [{ address }] = await signer.getAccounts();
+
+    accounts[name] = await StrideClient.create(RPC_ENDPOINT, signer, address, {
+      gasPrice: GasPrice.fromString("0.025ustrd"),
+      broadcastPollIntervalMs: 50,
+      resolveIbcResponsesCheckIntervalMs: 50,
+    });
+  }
+
+  console.log("waiting for chain to start...");
+  while (true) {
+    try {
+      const block =
+        await accounts.user.query.cosmos.base.tendermint.v1beta1.getLatestBlock(
+          {},
+        );
+
+      if (block?.block?.header.height! > 0n) {
+        break;
+      }
+
+      await sleep(50);
+    } catch (error) {}
+  }
+});
+
+// time variables in seconds
+const now = () => Math.floor(Date.now() / 1000);
+const minute = 60;
+const hour = 60 * minute;
+const day = 24 * hour;
+
+describe("x/airdrop", () => {
+  test("MsgCreateAirdrop", async () => {
+    const stridejs = accounts.admin;
+
+    const nowSec = now();
+    const airdropId = String(nowSec);
+
+    const msg =
+      stridejs.types.stride.airdrop.MessageComposer.withTypeUrl.createAirdrop({
+        admin: stridejs.address,
+        airdropId: airdropId,
+        rewardDenom: "ustrd",
+        distributionStartDate: fromSeconds(now()),
+        distributionEndDate: fromSeconds(nowSec + 3 * day),
+        clawbackDate: fromSeconds(nowSec + 4 * day),
+        claimTypeDeadlineDate: fromSeconds(nowSec + 2 * day),
+        earlyClaimPenalty: decToString(0.5),
+        allocatorAddress: stridejs.address,
+        distributorAddress: stridejs.address,
+        linkerAddress: stridejs.address,
+      });
+
+    const tx = await stridejs.signAndBroadcast([msg], 2);
+
+    if (tx.code !== 0) {
+      console.error(tx.rawLog);
+    }
+    expect(tx.code).toBe(0);
+
+    const { airdrop } = await stridejs.query.stride.airdrop.airdrop({
+      id: airdropId,
+    });
+
+    expect(airdrop!.id).toBe(airdropId);
+    expect(airdrop!.earlyClaimPenalty).toBe("0.5");
+  });
+});
+
+describe("ibc", () => {
+  test("MsgTransfer", async () => {
+    const stridejs = accounts.user;
+
+    const msg =
+      stridejs.types.ibc.applications.transfer.v1.MessageComposer.withTypeUrl.transfer(
+        {
+          sourcePort: "transfer",
+          sourceChannel: "channel-0",
+          token: coinFromString("1ustrd"),
+          sender: stridejs.address,
+          receiver: convertBech32Prefix(stridejs.address, "cosmos"),
+          timeoutHeight: {
+            revisionNumber: 0n,
+            revisionHeight: 0n,
+          },
+          timeoutTimestamp: BigInt(
+            `${Math.floor(Date.now() / 1000) + 3 * 60}000000000`, // 3 minutes from now as nanoseconds
+          ),
+          memo: "",
+        },
+      );
+
+    const tx = await stridejs.signAndBroadcast([msg]);
+    if (tx.code !== 0) {
+      console.error(tx.rawLog);
+    }
+    expect(tx.code).toBe(0);
+
+    const ibcAck = await tx.ibcResponses[0];
+    expect(ibcAck.type).toBe("ack");
+    expect(ibcAck.tx.code).toBe(0);
+  }, 30_000);
```

Comment:
is this the test timeout?

Reply:
yes

Reply:
default is 5s, which is not enough for tests with ibc operations